docs/walkthrough/index.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: default
+title: Walkthroughs & Tutorials
+---
+
+## Create a Resource Group
+
+Install the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) if you don't already have it.
+
+```bash
+az group create --name PromitorRG --location eastus
+```
+
+Output: 
+```json
+{ 
+  "id": "/subscriptions/<guid-subscription-id>/resourceGroups/PromitorRG",
+  "...": "..."
+}
+```
+
+## Create a Service Principal
+
+Use the resource group creation output to add a scope to your service principal:
+
+```bash
+az ad sp create-for-rbac \
+  --role="Monitoring Reader" \
+  --scopes="/subscriptions/<guid-subscription-id>/resourceGroups/PromitorRG"
+```
+
+Which should output something similar to
+
+```json
+{
+  "appId": "<guid-sp-app-id>",
+  "displayName": "azure-cli-2019-03-29-19-21-58",
+  "name": "http://azure-cli-2019-03-29-19-21-58",
+  "password": "<guid-sp-generated-password>",
+  "tenant": "<guid-tenant-id>"
+}
+```
+
+Save this output - the app ID, tenant ID, and password will be used later.
+
+## Create a Service Bus Namespace and Queue
+
+(From the [Service Bus Quickstart](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-cli))
+
+Before we install Promitor on AKS to start getting the metrics you want out of Azure Monitor, we need to set up the Azure Resource you want to monitor and have it ready to get metrics for (i.e Service Bus, Azure Functions, VM etc.). In this walkthrough we are going to set up a Service Bus queue as an example.
+
+First we'll need to create a namespace. Service Bus Namespaces need to be globally unique, so we won't use a default name in these commands.
+
+```bash
+az servicebus namespace create \
+  --resource-group PromitorRG \
+  --name <service-bus-namespace> \
+  --location eastus
+```
+
+We'll then create a queue in that namespace:
+
+```bash
+az servicebus queue create \
+  --resource-group PromitorRG \
+  --namespace-name <service-bus-namespace> \
+  --name demo_queue
+```
+
+Finally, get the connection string for this Service Bus namespace for use later.
+
+```bash
+az servicebus namespace authorization-rule keys list \
+  --resource-group PromitorRG \
+  --namespace-name <service-bus-namespace> \
+  --name RootManageSharedAccessKey \
+  --query primaryConnectionString \
+  --output tsv
+```
+
+## Create an AKS Cluster
+
+(From the [AKS Cluster Quickstart](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough#create-aks-cluster))
```

Comment:
Same here, interesting but less relevant - Either remove or add to a further reading section at the end